Initial working

gep13 opened the pull request.

gep13 committed:
Initial working

AdmiringWorm committed:
(maint) Corrected path seperator for nuspec file

AdmiringWorm commented:
We won't be able to test these changes on appveyor before the target branch have updated the appveyor image used to "Visual Studio 2019" (appveyor don't use the appveyor.yml file in a pr, only the one on the target branch).

AdmiringWorm committed:
(maint) Re-implement package creating

This allows us to use a single nuspec file for creating all
of the supported runtimes

AdmiringWorm commented:
I re-implemented the nuget package creation to better conform to creating packages for multiple runtimes (since we can use the same nuspec file for each runtime).

gep13 commented:
@AdmiringWorm thank you very much for taking a look at this!  The refactoring that you have done is a stroke of genius!  I have rebased this branch on the latest develop, which is now running on the right AppVeyor base image, and the build is now working.  There is still a little bit of re-work to be done here, but basically it seems to be working as expected.
